manifests/spoke/spoke-clusterrole.yaml

```diff
@@ -0,0 +1,33 @@
+# Clusterrole for work agent in addition to admin clusterrole.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:open-cluster-management:{{ .SpokeCoreName }}-agent
+rules:
+# Allow agent to get/list/watch/create/delete crds.
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: ["customresourcedefinitions"]
+  verbs: ["get", "list", "watch", "create", "delete", "update"]
+# Allow agent to create/delete namespaces, get/list are contained in admin role already
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["create", "delete"]
+# Allow agent to managem role/rolebinding/clusterrole/clusterrolebinding
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["clusterrolebindings", "rolebindings"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["clusterroles", "roles"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete", "escalate", "bind"]
+# Allow agent to get/list/watch nodes.
+- apiGroups: [""]
+  resources: ["nodes"]
```

Comment:
I wouldn't expect the operator to be able to see nodes. Is this a role for an operator or an operand?

Reply:
registraion controller need this to collect capacity and allocatable resources of all nodes